src/pyodide/internal/util.ts

```diff
@@ -1,3 +1,8 @@
+export function reportError(e: any): never {
+  e.stack?.split('\n').forEach((s: any) => console.warn(s));
```

Comment:
Can you add a comment to here explaining why we are splitting the stack and printing it seperately? Particularly the limitation of test runner etc.

```suggestion
  e.stack?.split('\n').forEach((s: string) => console.warn(s));
```